File: Back/InsurancesAPI/Models/Business/Insurance.cs
using Models.Enums;
using System.ComponentModel.DataAnnotations;

namespace Models.Business
{
    public class Insurance
    {
        [Key]
        public long InsuranceID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public InsuranceTypeEnum Type { get; set; }
        public decimal Coverage { get; set; }
    }
}

File: Front/GAPInsurancesApp/src/app/models/insurance.ts
import { InsuranceTypeEnum } from "../framework/enum/insurance-enum";
import { Type } from "@angular/compiler";

export class InsuranceModel{
    InsuranceID: number;
    Name: string;
    Description: string;
    Type: InsuranceTypeEnum;
    Coverage: number;

    InsuranceModel(){
        this.InsuranceID = -1;
        this.Name = "";
        this.Description = "";
        this.Type = InsuranceTypeEnum.None;
    }


}

